node_modules/oae-core/upload/js/upload.js

```diff
@@ -262,6 +262,9 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
                 }
             }, $('#upload-selected-container', $rootel));
 
+            // Hide the spinner icons
```

Comment:
We don't have to explicitly hide the icon since the parent `<div>` is hidden. (I probably just missed that before.) I'll remove the code.